gtsfm/two_view_estimator.py

```diff
@@ -250,13 +250,18 @@ def run_2view(
         putative_corr_idxs: np.ndarray,
         camera_intrinsics_i1: Optional[gtsfm_types.CALIBRATION_TYPE],
         camera_intrinsics_i2: Optional[gtsfm_types.CALIBRATION_TYPE],
-        im_shape_i1: Tuple[int, int],
-        im_shape_i2: Tuple[int, int],
         i2Ti1_prior: Optional[PosePrior],
         gt_wTi1: Optional[Pose3],
         gt_wTi2: Optional[Pose3],
         gt_scene_mesh: Optional[Any] = None,
-    ) -> Tuple[Optional[Rot3], Optional[Unit3], np.ndarray, Dict[str, Optional[TwoViewEstimationReport]]]:
+    ) -> Tuple[
+        Optional[Rot3],
+        Optional[Unit3],
+        np.ndarray,
+        TwoViewEstimationReport,
+        TwoViewEstimationReport,
+        Optional[TwoViewEstimationReport],
```

Comment:
As far as I can tell, this last return arg should not be optional, and the docstring seems wrong in the ISP, since we never return a None report? https://github.com/borglab/gtsfm/blob/master/gtsfm/frontend/inlier_support_processor.py#L45

Reply:
thanks for catching that, it was annoying to have that optional around everywhere.